templates/common/kubevirt/files/002-nmstate-arp-proxy-ipv6-gw.yaml

```diff
@@ -0,0 +1,11 @@
+path: "/etc/nmstate/002-nmstate-arp-proxy-ipv6-gw.yml"
+contents:
+  inline: |
+    capture:
+      ethernet-nics: interfaces.type=="ethernet"
+    desiredState:
+      routes:
+        config:
+        - destination: ::/0
+          next-hop-interface: "{{`{{ capture.ethernet-nics.interfaces.0.name }}`}}"
+          next-hop-address: d7b:6b4d:7b25:d22f::1
```

Comment:
is this a dummy/null route IPV6 value?

Reply:
This is a link local IPv6 refering to some arp proxy configuration at ovn, you can see how it works at the [enhancement](https://github.com/openshift/enhancements/blob/master/enhancements/network/ovn-hypershift-live-migration.md)

Reply:
If it's an IPv6 link-local address, it should be in the range of fe80::/10, and that address you've specified is clearly not in that range.